mysql-test/suite/sys_vars/r/innodb_encrypt_tables_basic.result

```diff
@@ -20,6 +20,8 @@ SET GLOBAL innodb_encrypt_tables=1.1;
 ERROR 42000: Incorrect argument type to variable 'innodb_encrypt_tables'
 SET GLOBAL innodb_encrypt_tables=1e1;
 ERROR 42000: Incorrect argument type to variable 'innodb_encrypt_tables'
+SET GLOBAL innodb_encrypt_tables=NULL;
```

Comment:
Like yura said, how did this test even run? shouldn't we get 'unknown variable error'?

Reply:
Hmm it does error out:

=============================================================================
                  TEST NAME                       RESULT  TIME (ms) COMMENT
------------------------------------------------------------------------------
[100%] sys_vars.innodb_encrypt_tables_basic      [ fail ]
        Test ended at 2019-08-20 09:31:40

CURRENT_TEST: sys_vars.innodb_encrypt_tables_basic
mysqltest: At line 1: Query 'SET @@GLOBAL.innodb_encrypt_tables="OFF"' failed.
ERROR 1193 (HY000): Unknown system variable 'innodb_encrypt_tables'
safe_process[10967]: Child process: 10968, exit: 1

I guess this change is brought by cherry-pick

Reply:
Please remove both the test and .result files